package.json

```diff
@@ -110,7 +110,7 @@
     "react-native-svg-transformer": "^0.14.3",
     "react-test-renderer": "^17.0.1",
     "shell-utils": "^1.0.10",
-    "typescript": "4.3.2"
+    "typescript": "4.5.0-dev.20210810"
```

Comment:
Do we have to update to a dev version of typescript? 
I prefer to be in a stable version, do literals not supported in v4.3.2?

Reply:
Yes it does not support

Reply:
I am Chinese, because there will be a time difference, so the reply will be slower, please forgive me

Reply:
no worries. 
If it's ok, I prefer to wait for a stable version of Typescript first that supports literals

Reply:
ok

Reply:
I see that version 4.4.2 has been published recently
Do you think it contains literals?

Reply:
i was try it,it works!

Reply:
I upgrade typescript version to 4.4.2. Then rebuild the typings.
And many files have been modified, but I think that is ok, please confirm

Reply:
I upgrade typescript version to 4.4.2. Then fixed conflect